app/bundles/CoreBundle/Assets/js/4.builder.js

```diff
@@ -4,10 +4,10 @@
  * @returns {string}
  */
 Mautic.getUrlParameter = function (name) {
-    name = name.replace(/[\[]/, '\\[').replace(/[\]]/, '\\]');
-    var regex = new RegExp('[\\?&]' + name + '=([^&#]*)');
-    var results = regex.exec(location.search);
-    return results === null ? '' : decodeURIComponent(results[1].replace(/\+/g, ' '));
+  name = name.replace(/[\[]/, '\\[').replace(/[\]]/, '\\]');
```

Comment:
We use four spaces, not two, for indentions. Looks like this PR changed them all. Let's revert that. Thanks!